### PR TITLE
Codeowners tweak

### PR DESCRIPTION
**Pull request by mitschabaude**

### CODE REVIEW

**Comment by mitschabaude** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

File: CODEOWNERS [hunk 1]
```diff
@@ -30,8 +30,8 @@
 /src/lib/ @MinaProtocol/protocol-eng-reviewers
 
 /src/lib/blake2/ @MinaProtocol/crypto-eng-reviewers
-/src/lib/cli_lib/ @MinaProtocol/product-eng-reviewers
-/src/lib/mina_commands @MinaProtocol/product-eng-reviewers
+/src/lib/cli_lib/ @MinaProtocol/protocol-eng-reviewers
+/src/lib/mina_commands @MinaProtocol/protocol-eng-reviewers
```
Comment (deepthiskumar) [inline CODEOWNERS:34]: I think we can remove these lines given protocol-eng owns `src/lib/` by default

Comment (mitschabaude) [reply CODEOWNERS:34]: ah right!

**Comment by mitschabaude** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Pull request merged**